test/e2e/specs/wp-gutenberg-experimental-features-spec.js

```diff
@@ -23,10 +23,21 @@ const gutenbergUser =
 // of the @wordpress/* packages. The purpose of these tests is to give us an early
 // warning if an experimental feature has been removed or renamed.
 const EXPERIMENTAL_FEATURES = {
```

Comment:
It's unclear whether and when this is actually run. I can see the spec file in CircleCI logs, but I don't see the describe actually appear like it isn't being run. We may need to tag this describe somehow, e.g. `@parallel` and/or whatever tag runs on Gutenberg edge tests.

@fullofcaffeine can own that bit? I'd expect this to run as part of the full suite and the Gutenberg edge suites.

Reply:
> @fullofcaffeine can own that bit? 

Sure! Tracked it [here](https://github.com/orgs/Automattic/projects/185#card-50140939). Will get to it as soon as I can.